CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Current Master
 
+- Add an extra px to address for Firefox clipping.
+- Adjust max height for account detail section for Firefox.
```

Comment:
Would prefer changelog entry that is meaningful to users, like: "Fixes scrolling in firefox".